staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -129,11 +129,11 @@ func DefaultWatchErrorHandler(r *Reflector, err error) {
 		// Don't set LastSyncResourceVersionUnavailable - LIST call with ResourceVersion=RV already
 		// has a semantic that it returns data at least as fresh as provided RV.
 		// So first try to LIST with setting RV to resource version of last observed object.
-		klog.V(4).Infof("%s: watch of %v closed with: %v", r.name, r.expectedTypeName, err)
+		klog.Errorf("%s: watch of %v closed with: %v", r.name, r.expectedTypeName, err)
```

Comment:
With klog/v2, may be ErrorS more available.
```suggestion
		klog.ErrorS(err,"watch closed", "name",r.name, "expectedTypeName",r.expectedTypeName)
```